core/src/main/java/hudson/model/ParametersDefinitionProperty.java

```diff
@@ -171,7 +171,7 @@ public void _doBuild(StaplerRequest req, StaplerResponse rsp, @QueryParameter Ti
         }
 
     	WaitingItem item = Jenkins.getInstance().getQueue().schedule(
-                getJob(), delay.getTime(), new ParametersAction(values), new CauseAction(new Cause.UserIdCause()));
+                getJob(), delay.getTimeInSeconds(), new ParametersAction(values), new CauseAction(new Cause.UserIdCause()));
```

Comment:
Should it be `getTimeInMillis()`? Otherwise you feed seconds to the milliseconds method here and below

Reply:
`schedule` and similar methods take arguments measured in seconds so I see nothing wrong here.

Reply:
I was just surprised, because it was passing milliseconds before. I should have read the issue